frontend/src/modules/vehicles/VehicleLocation.vue

```diff
@@ -2,22 +2,18 @@
   <div>
     <CCard>
       <CCardBody>
-        <h4>Location</h4>
+        <CRow class="m-0 d-flex justify-content-between align-items-center" v-if="vehicle.zone">
+          <h4>Location</h4>
+          <p class="m-0">{{ vehicle.zone.name }}</p>
```

Comment:
This line of code was causing issue but it was fixed above by adding the v-if to check if zone is defined